packages/orchestration/src/exos/local-orchestration-account.js

```diff
@@ -87,7 +88,9 @@ export const prepareLocalOrchestrationAccountKit = (
     {
       holder: HolderI,
       undelegateWatcher: M.interface('undelegateWatcher', {
-        onFulfilled: M.call([M.splitRecord({ completionTime: M.string() })])
+        onFulfilled: M.call([
+          M.splitRecord({ completionTime: TimestampProtoShape }),
```

Comment:
Using `splitRecord` here to make this forward compatible (cosmos v0.50 adds `amount: Coin` to the response: https://buf.build/cosmos/cosmos-sdk/docs/main:cosmos.staking.v1beta1#cosmos.staking.v1beta1.MsgUndelegateResponse)

Are there other places we might need to consider this approach? Should we be guarding watcher's more loosely?

Reply:
I guess the local-orch-account is less likely to get an unexpected upgrade to v0.50, more the cosmos-orch-account. We do not have something similar in those interface guards since the value is still a base64 byte string at that point.

Reply:
Good question tho. Thanks for raising and resolving.